stubs/Helpers.stub

```diff
@@ -10,6 +10,13 @@
  *
  * @throws \Exception
  */
-function retry($times, callable $callback, $sleep = 0, $when = null)
-{
-}
+function retry($times, callable $callback, $sleep = 0, $when = null){}
+
+/**
+ * @template TModel
+ * @template TAmount
+ * @param  class-string<TModel>  $class
+ * @param  TAmount  $amount
+ * @return \Illuminate\Database\Eloquent\FactoryBuilder<TModel, TAmount>
```

Comment:
does phpstan have support for template conditionals? I solved for these specific types over at psalm with the following https://github.com/mr-feek/psalm-plugin-laravel/pull/1/files#diff-3a21265d8cec9058a791ab9d9a40cb5aR39